src/app/components/forms/user/block/block.ts

```diff
@@ -1,26 +1,42 @@
 import { Component } from 'vue-property-decorator';
 import { Api } from '../../../../../_common/api/api.service';
+import AppFormControlToggle from '../../../../../_common/form-vue/control/toggle/toggle.vue';
 import { BaseForm, FormOnSubmit } from '../../../../../_common/form-vue/form.service';
 import { Growls } from '../../../../../_common/growls/growls.service';
 
 type BlockData = {
 	username: string;
+	removeShouts: boolean;
+	removePostComments: boolean;
 };
 
 @Component({
-	components: {},
+	components: {
+		AppFormControlToggle,
+	},
 })
 export default class FormUserBlock extends BaseForm<BlockData> implements FormOnSubmit {
 	resetOnSubmit = true;
 
 	async onSubmit() {
 		const response = await Api.sendRequest(`/web/dash/blocks/add`, this.formModel);
 		if (response.success) {
```

Comment:
Hey, I just noticed this. I think you're supposed to do just the API call in onSubmit, and then you should make a new onSubmitSuccess method for handling the success and you can push all this into that.